araproc/analysis/standard_reco.py

```diff
@@ -819,3 +821,93 @@ def min_frac_corr_depth_multiple(self, *maps, fraction=0.6, z_thresh=0):
             raise RuntimeError("No maps meet the fractional correlation threshold within the z_thresh.")
 
         return min_depth, min_depth_index
+
+    def get_corr_snr(self, ch1, ch2, wave_packet):
+
+        """
+        Calculates channel-pair correlation SNR
+    
+        Parameters
+        ----------
+        ch1: int
+            First channel number for correlation.
+        ch2: int
+            Second channel number for correlation.
+        wavepacket: dict
+            A dict with three entries:
+              "event": int  
+                Event number
+              "waveforms": dict
+                Dict mapping RF channel ID to waveforms.
+                Keys are channel id (an integer)
+                Values are TGraphs
+              "trace_type": string
+                Waveform type requested by which_trace.
+ 
+        Returns
+        -------
+        corr_snr: float
+           Channel pair correlation SNR.
+        """
+
+        corr_func = self.__get_correlation_function(ch1, ch2, wave_packet)
+        corr_snr = hsnr.get_hill_snr(corr_func)
```

Comment:
Good call with this. I think that's the right way to do it since this is hilbert enveloped.